BlueEastCode/bluerain-plugin-apollo
# Mutation Component added

## What I did
Added Mutation component from upcoming react-apollo 2.1.

## How to test

Is this testable with jest or storyshots?
It may not be required as there is no custom code.

Are any of the existing tests are failing ?
No.

Does this need an update to the documentation?
Yes

Does it have breaking changes ?
No

If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
